src/HealthChecks.Uris/UriHealthCheck.cs

```diff
@@ -55,6 +55,15 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
                             return new HealthCheckResult(context.Registration.FailureStatus, description: $"Discover endpoint #{idx} is not responding with code in {expectedStatusCodes.Min}...{expectedStatusCodes.Max} range, the current status is {response.StatusCode}.");
                         }
 
+                        if (!string.IsNullOrWhiteSpace(item.ExpectedContent))
+                        {
+                            var responseBody = await response.Content.ReadAsStringAsync();
+                            if (responseBody != item.ExpectedContent)
+                            {
+                                return new HealthCheckResult(context.Registration.FailureStatus, description: $"The expected value '{item.ExpectedContent}' was not found in the reponsebody!");

```

Comment:
```suggestion
                                return new HealthCheckResult(context.Registration.FailureStatus, description: $"The expected value '{item.ExpectedContent}' was not found in the response body!");

```

Reply:
isn't responsebody one word?

Reply:
Than it should be responsebody, not reponsebody.

Reply:
completely missed the 'S', my bad! I'll fix that right away.

Reply:
made an update to the pull-request with the fixed spelling.